### ISSUE TITLE
array_splice_reverse redeclared

### ISSUE THREAD

**Issue by rafacouto** [mpdf/mpdf#501]
With PHP 7.1.8 and 7.0.0-RC1 composer package:

```
Fatal error: Cannot redeclare array_splice_reverse() (previously declared
in /app/drupal/vendor/mpdf/mpdf/src/functions.php:11) 
in /app/drupal/vendor/mpdf/mpdf/src/functions.php on line 15
```

Is functions.php required twice from anywhere?

**Comment by finwe** [comment]
No, unless you require the composer autoload twice, I guess. Can you elaborate and debug further?

**Comment by rafacouto** [comment]
I will search for all requires/includes composer-autoloads from drupal and report further info. Anyway, functions.php is loaded just by composer and it should be require/include "once" the files. Another fact is php-7.0.22 with mpdf-6.1.4 is going right. Thank you for replying 👍

**Comment by finwe** [comment]
Isn't a function of a same name defined somewhere in Drupal, outside mPDF code? Anyway, it seems that the function is not used anywhere in the mPDF code, so I guess it could be removed.

**Comment by rafacouto** [comment]
Another idea is to wrap those functions at functions.php within a FunctionsHelper class and refactor the code to use the new class. It will be more php/composer aware...

**Comment by finwe** [comment]
Resolved in cf13268.

**Comment by rafacouto** [comment]
v7.0.0-RC2 is running OK without the reported error. Thank you for the bugfix 👍
